go.mod

```diff
@@ -8,7 +8,6 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/stretchr/testify v1.2.2
-	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad // indirect
-	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
-	golang.org/x/tools v0.1.2 // indirect
+	golang.org/x/net v0.0.0-20190611141213-3f473d35a33a
```

Comment:
I don't understand why the downgrade of this

Reply:
`Go mod` picked it up

Reply:
And we're fine with it?